cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -214,6 +214,9 @@ parameters:
 - name: CPU_LIMIT
   description: CPU limit.
   value: "1"
+- name: CS_MI_CLIENT_ID
```

Comment:
Should we prefix this with `AZURE`? For cloud specific parameters, I think it's prefixed with the associated cloud, e.g. `AWS_STS_POLICY_DIRECTORY`.  

Can also be done in a follow up PR if we do decide if we want to add this.

Reply:
sure. fixed